CHANGELOG.md

```diff
@@ -6,14 +6,13 @@ The format is based on [Keep a Changelog][keepachangelog] and this project adher
 
 ## UNRELEASED
 
-### Added
+### Changed
 
-- Class `ReportMakeParams` to build make-report parameters
-- Optional parameter `idempotenceKey` for report-make requests
+- Refactory `ClientInterface` & `Client` class
 
-### Changed
+### Added
 
-- Method `ClientInterface::userReportMake` now takes object `ReportMakeParams` as parameter
+- Optional parameter `idempotenceKey` in `ReportMakeParams` for report-make requests
```

Comment:
All is ok here?

Reply:
Hmmm... I think yes. Am I missing something?

Reply:
I think yes. Am I missing something?

Reply:
It seems like you forgot to describe **all** changes and broke down previous correct changelog lines

Reply:
CHANGELOG.md was fixed 3651963cbe79b46b6f5787b9dfea4ac096db7320